openshift/cirras-underwriting-jasper-deployment.yaml

```diff
@@ -32,22 +54,16 @@ spec:
               memory: 400Mi 
           image: >-
             docker.io/bitnami/mariadb:latest
+          command:
+            ['sh', '-c']
+          args:
+            ['. /vault/secrets/env && catalina.sh run']
```

Comment:
I don't see a docker file in cirras-underwriting-reports folder

Reply:
There isn't a Dockerfile for it. The deployment loads the image specified in `spec.template.spec.containers.image`, which in this case is pointing at DockerHub. We'd have to find documentation on how to set or override the ENTRYPOINT and/or CMD for each Image. Not sure if it's worthwhile since we won't be using Jasper Server anymore.